test/Microsoft.Framework.Configuration.Binder.Test/ConfigurationCollectionBindingTests.cs

```diff
@@ -182,9 +183,9 @@ public void StringDictionaryBinding()
 
             Assert.Equal(3, options.StringDictionary.Count);
 
-            Assert.Equal("val_1", options.StringDictionary["abc"]);
-            Assert.Equal("val_2", options.StringDictionary["def"]);
-            Assert.Equal("val_3", options.StringDictionary["ghi"]);
+            Assert.Equal("val_1", options.StringDictionary["StringDictionary:abc"]);
```

Comment:
Or maybe its a bug fix?  Either way this type of change in tests looks suspicious :)